packages/application-generic/src/factories/sms/handlers/africas-talking.handler.ts

```diff
@@ -0,0 +1,24 @@
+import { ChannelTypeEnum } from '@novu/shared';
+import { AfricasTalkingSmsProvider } from '@novu/africas-talking';
+import { ICredentials } from '@novu/dal';
+import { BaseSmsHandler } from './base.handler';
+
+export class AfricasTalkingSmsHandler extends BaseSmsHandler {
+  constructor() {
+    super('africastalking', ChannelTypeEnum.SMS);
```

Comment:
```suggestion
    super('africas-talking', ChannelTypeEnum.SMS);
```
Based on the enum value set.